lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -102,6 +102,10 @@ def connected?
             !!@requestor
           end
 
+          def supports_metadata?
+            version(nil) >= '2.1.5'
+          end
```

Comment:
String comparison of versions can break down.  See [this](https://medium.com/little-programming-joys/how-to-compare-version-strings-in-ruby-ced0916a4b71)